src/main/java/frc/robot/Constants.java

```diff
@@ -117,4 +117,6 @@
     public static final double ballFlywheelratio = 2;
     public static final int shooterCargoID = 9;
     public static final double cargoRotation = 3;
+
+    public static final double GearRatioOut = 0; //Idk if need But It fix errors in Shooter Sub
```

Comment:
gearRatioOut exists already on line 114. Rename other references to that (lowerCamelCase)